app/helpers/sort_helper.rb

```diff
@@ -214,6 +216,12 @@ def sort_clause
     @sort_criteria.to_sql
   end
 
+  # Determines whether the current selected sort criteria
+  # is identical to the default
+  def default_sort_order?
+    @sort_default == @sort_criteria.criteria
```

Comment:
Are you sure that this works as you intend?

Just asking because I recently (today?) found out, that `criteria=` does **transform** the value before being stored. Quite harshly... And if I see this correctly `@sort_default` stores the pre-transformation value...